ui/packages/ui/src/app/pages/connectors/ConnectorsPage.tsx

```diff
@@ -47,7 +47,7 @@ export const ConnectorsPage: React.FunctionComponent = (props) => {
       return conn.name;
     });
     history.push({
-      pathname: "/app/create-connector",
+      pathname: "/create-connector",
```

Comment:
maybe `/#create-connector`?

Reply:
yes, it will be appended to the basename. so it will be like `#app//create-connector`